tracing-fmt/src/default.rs

```diff
@@ -231,14 +246,27 @@ where
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let mut seen = false;
-        let style = Style::new().bold();
+        let style = if self.1 {
+            Some(Style::new().bold())
+        } else {
+            None
```

Comment:
Alternatively, instead of using an `Option`, we could just use `Style::new()` in this case, which I believe should not apply any formatting. That way, we wouldn't need the redundant `if let`s later.

But, this is not relevant if we end up changing the format impls to be statically dispatched as I mentioned above.